mimic/canned_responses/fastly.py

```diff
@@ -0,0 +1,278 @@
+            u'readonly': '',
+            u'ip_whitelist': u'0.0.0.0/0',
+            u'owner_id': u'asewQWEwdRnLnCKyMztZqkQvy',
+            u'phone_number': u'770-123-1749',
+            u'postal_address': None,
+            u'billing_ref': None,
+            u'can_reset_passwords': True,
+            u'has_improved_security': u'1',
+            u'stripe_account': None,
+            u'name': u'Poppy - Test',
+            u'created_at': u'2014-11-03T23:37:43+00:00',
+            u'can_stream_syslog': u'1',
+            u'pricing_plan': u'developer',
+            u'billing_contact_id': None,
+            u'has_streaming': u'1'}
+        return current_customer
+
+    def create_service(self, url_data):
+        """
+        Returns POST service with response json.
+
+        :return: a JSON-serializable dictionary matching the format of the JSON
+                 response for fastly_client.create_service()
+                 ("/service") request.
+        """
+        data = dict((key, value[0]) for key, value in url_data)
+
+        publish_key = uuid.uuid4().hex
+        service_id = uuid.uuid4().hex
+        service_name = data['name']
+
+        self.fastly_cache[service_name] = {
+            'service_details': {
+                u'comment': '',
+                u'locked': False,
+                u'updated_at': u'2014-11-13T14:29:10+00:00',
+                u'created_at': u'2014-11-13T14:29:10+00:00',
+                u'testing': None,
+                u'number': 1,
+                u'staging': None,
+                u'active': None,
+                u'service_id': service_id,
+                u'deleted_at': None,
+                u'inherit_service_id': None,
+                u'deployed': None},
+            'service_name': service_name
+        }
+        self.fastly_cache[service_id] = self.fastly_cache[service_name]
+
+        create_service = {
+            u'comment': '',
+            u'publish_key': publish_key,
+            u'name': service_name,
+            u'versions': [{u'comment': '', u'locked': u'0',
+                           u'service': service_id,
+                           u'updated_at': u'2014-11-12T18:43:21',
+                           u'created_at': u'2014-11-12T18:43:21',
+                           u'testing': None, u'number': u'1',
+                           u'staging': None,
+                           u'active': None,
+                           u'service_id': service_id,
+                           u'deleted_at': None,
+                           u'inherit_service_id': None,
+                           u'deployed': None,
+                           u'backend': 0}],
+            u'created_at': u'2014-11-12T18:43:21+00:00',
+            u'updated_at': u'2014-11-12T18:43:21+00:00',
+            u'customer_id': data['customer_id'],
+            u'id': service_id}
+        return create_service
+
+    def get_service_by_name(self, service_name):
+        """Returns service details json.
```

Comment:
Could you write slightly more descriptive docstrings here?  I don't know what "service details json" means.